test/internal/constants/images.go

```diff
@@ -0,0 +1,54 @@
+package constants
+
+// not technically constants, but close enough ...
+
+import (
+	"fmt"
+)
+
+const (
+	DockerImageRepo     = "docker.io/library"
+	McrWindowsImageRepo = "mcr.microsoft.com/windows"
+
+	ImageLinuxAlpineLatest = "docker.io/library/alpine:latest"
+	ImageLinuxPause31      = "k8s.gcr.io/pause:3.1"
+)
+
+var (
```

Comment:
Should we share this so that we can replace the functions in `test/cri-containerd/main_test.go`?

Reply:
I updated the `getWindowsServerCoreImage` in `test/cri-containerd/main_test.go` to use this